pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -438,17 +443,27 @@ func requestHasParameters(req *autoscaling.ProvisioningRequest, prc *kueue.Provi
 	if req.Spec.ProvisioningClassName != prc.Spec.ProvisioningClassName {
 		return false
 	}
-	if len(req.Spec.Parameters) != len(prc.Spec.Parameters) {
-		return false
-	}
-	for k, vReq := range req.Spec.Parameters {
-		if vCfg, found := prc.Spec.Parameters[k]; !found || vReq != autoscaling.Parameter(vCfg) {
+	for k, vCfg := range prc.Spec.Parameters {
```

Comment:
could you update the function name and/or add a comment please? It is not clear from the function name what this check is doing.

especially now that we'r no longer just checking for equality of the two maps.

Reply:
Of course, fixed!